### ISSUE TITLE
Can not install ohpc-gnu12-runtimes on Rocky 8

### ISSUE THREAD

**Issue by zhaoqi9** [openhpc/ohpc#1668]
Refer to the content of link https://github.com/openhpc/ohpc/issues/1644
I also encountered the same problem, but according to the installation content of ohpc-gnu9-runtimes.x86_64, I guess ohpc-gnu12-runtimes.x86_64 is also the same, so I manually installed charliecloud-ohpc and singularity-ohpc.

**Comment by adrianreber** [comment]
Sorry, but I do not understand what you are trying to report. Can you rephrase it maybe? Or show some output of the error you are seeing?

**Comment by zhaoqi9** [comment]
Sorry, I didn't express myself clearly.
I encountered the following error and referred to https://github.com/openhpc/ohpc/issues/1644. 

# yum install ohpc-gnu12-runtimes
Last metadata expiration check: 1:54:10 ago on Tue 31 Jan 2023 03:44:53 PM CST.
Error:
 Problem: cannot install the best candidate for the job
  - nothing provides singularity needed by ohpc-gnu12-runtimes-2.6-7.1.ohpc.2.6.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)

According to the installation content of ohpc-gnu9-runtimes.x86_64, I guess ohpc-gnu12-runtimes.x86_64 is the same, so I manually installed charliecloud-ohpc and singularity-ohpc. When installing ohpc-gnu12-runtimes.x86_64 with the above errors, I'm not sure if this method is correct.

**Comment by zhaoqi9** [comment]
Sorry, I didn't express myself clearly.
I encountered the following error and referred to https://github.com/openhpc/ohpc/issues/1644.

```
[root@sms ~]# yum install ohpc-gnu12-runtimes
Last metadata expiration check: 1:54:10 ago on Tue 31 Jan 2023 03:44:53 PM CST.
Error:
 Problem: cannot install the best candidate for the job
  - nothing provides singularity needed by ohpc-gnu12-runtimes-2.6-7.1.ohpc.2.6.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

According to the installation content of ohpc-gnu9-runtimes.x86_64, I guess ohpc-gnu12-runtimes.x86_64 is the same, so I manually installed charliecloud-ohpc and singularity-ohpc. When installing ohpc-gnu12-runtimes.x86_64 with the above errors, I'm not sure if this method is correct.

**Comment by adrianreber** [comment]
This sounds identical to #1644 

You can either download a singularity package from somewhere (see https://github.com/openhpc/ohpc/issues/1644#issuecomment-1370821241) or wait for the fix we want to distribute in 2.6.1.

**Comment by zhaoqi9** [comment]
Many thanks.
